src/components/Footer.tsx

```diff
@@ -0,0 +1,114 @@
+import Link from 'next/link';
+import { Navigation } from '@/enums/navigation.enum';
+import { SOCIAL_PROVIDER_REACT_ICONS } from '@/constants/social-provider-react-icons';
+
+type FooterSectionType = {
+	section: string;
+	links: { label: string; href: string }[];
+};
+
+const FooterSection = ({ section, links }: FooterSectionType) => (
+	<div className="flex flex-col space-y-3">
+		<h1 className="text-xl font-bold opacity-50 text-white">{section}</h1>
+		<ul className="flex flex-col space-y-2 font-semibold text-white">
+			{links.map((link) => (
+				<li key={link.label}>
+					<Link
+						className="hover:opacity-60 transition duration-200"
+						href={link.href}
+					>
+						{link.label}
+					</Link>
+				</li>
+			))}
+		</ul>
+	</div>
+);
+
+const SocialIcons = () => (
+		<div className="inline-flex gap-x-5 text-2xl">
+				{SOCIAL_PROVIDER_REACT_ICONS.map(provider => (
+
+					<Link
+					key={provider.provider}
```

Comment:
sometimes you use index and sometimes names

Reply:
if there are unique names and no more will be needed, it is better to use the name in this context